ChangeLog

```diff
@@ -48,7 +73,7 @@
           options.
         * Additional unit tests.
         * Reported bugs fixed.
-        * Example applications help reports on (D)TLS library capabilities
```

Comment:
Hm, do we really want to change old release notes?

Reply:
OK - went back to the original and pushed code.